.github/ISSUE_TEMPLATE/repo-archive-application.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Repository Archival
+about: Retire and archive a KubeEdge repository
+title: "REQUEST: Archive repo <repo>"
+---
+
```

Comment:
@fisherxu if i had a request for the archiving the repo, probably i would like to know `Why archiving is required?` as 1st, and probably it would be good summarize for later use. just a suggestion.

Reply:
Good idea, and whether the third item `### 3. **Additional context for request.**` can include `Why archiving is required?`? It will include the context and discussion result within the SIG.